### DISCUSSION TITLE
Every other day

### DISCUSSION THREAD

**Discussion by Dsfwy** [iSoron/uhabits#715] [Help & FAQ] [answered]
How do I set up a habit for every other day? Thanks!

**Comment by Dsfwy** [comment]
Never mind, I figured it out right after I posted this, of course. Sorry!

**Comment by iSoron** [comment, marked as answer]
For future reference, in the current version of the app (Loop 1.8), you can set a habit for every other day by specifying "1 time in 2 days". In Loop 2.0, you will be able to select the more intuitive option of "Every 2 days".
